lib/pages/add_expense.dart

```diff
@@ -20,6 +23,7 @@ class _AddExpensePageState extends State<AddExpensePage> {
 
   @override
   Widget build(BuildContext context) {
+    final expenseProvider = Provider.of<ExpenseProvider>(context);
     return Scaffold(
         appBar: AppBar(
           actions: [
```

Comment:
Appbar adds a back button by default, actions appear on the right so the right hand arrow can be removed.